cmd/dagger/main.go

```diff
@@ -166,6 +167,23 @@ var rootCmd = &cobra.Command{
 
 		return nil
 	},
+	PersistentPostRun: func(cmd *cobra.Command, args []string) {
+		highlight := lipgloss.NewStyle().
+			Bold(true).
+			Underline(true)
+		_, _, ok := enginetel.URLForTrace(cmd.Context())
+		switch {
+		case !ok &&
+			cmd.Flags().Changed("verbose") ||
+			cmd.Flags().Changed("debug") ||
+			cmd == moduleInitCmd:
+			fmt.Fprintf(cmd.ErrOrStderr(), `
+Log-in or create an account to visualize %s in Dagger Cloud:
+https://dagger.cloud/signup?quickstart=true
+
+`, highlight.Render(`pipeline traces`))
```

Comment:
Just noticed this now - this should be `!ok && (...)` - since `&&` has a higher precedence, it means that even when logged in, the message is still shown for `debug` and `moduleInitCmd`.

Reply:
noted, thx for reporting this one Justin <3